nohup: ignoring input
W0401 13:08:13.954348 140548822046528 torch/distributed/run.py:779] 
W0401 13:08:13.954348 140548822046528 torch/distributed/run.py:779] *****************************************
W0401 13:08:13.954348 140548822046528 torch/distributed/run.py:779] Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
W0401 13:08:13.954348 140548822046528 torch/distributed/run.py:779] *****************************************
/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/sklearn/metrics/_classification.py:1471: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 due to no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/sklearn/metrics/_classification.py:1471: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 due to no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/sklearn/metrics/_classification.py:1471: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 due to no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/sklearn/metrics/_classification.py:1471: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 due to no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
/home/ubuntu/tenerife/data/ZZ_githubRepos/baselinesLungAmbition/ImageModels/EfficientNet/EfficientNet3D/model/metrics.py:20: RuntimeWarning: invalid value encountered in scalar divide
  test_NPV=tn/(tn+fn)
/home/ubuntu/tenerife/data/ZZ_githubRepos/baselinesLungAmbition/ImageModels/EfficientNet/EfficientNet3D/model/metrics.py:20: RuntimeWarning: invalid value encountered in scalar divide
  test_NPV=tn/(tn+fn)
/home/ubuntu/tenerife/data/ZZ_githubRepos/baselinesLungAmbition/ImageModels/EfficientNet/EfficientNet3D/model/metrics.py:20: RuntimeWarning: invalid value encountered in scalar divide
  test_NPV=tn/(tn+fn)
/home/ubuntu/tenerife/data/ZZ_githubRepos/baselinesLungAmbition/ImageModels/EfficientNet/EfficientNet3D/model/metrics.py:20: RuntimeWarning: invalid value encountered in scalar divide
  test_NPV=tn/(tn+fn)
[rank1]: [rank1]: Traceback (most recent call last):
[rank1]: [rank1]:   File "mainBMY0.py", line 194, in <module>
[rank1]: [rank1]:     model = DDP(model, device_ids=[rank], find_unused_parameters=True)
[rank1]: [rank1]:   File "/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/torch/nn/parallel/distributed.py", line 822, in __init__
[rank1]: [rank1]:     _verify_param_shape_across_processes(self.process_group, parameters)
[rank1]: [rank1]:   File "/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/torch/distributed/utils.py", line 286, in _verify_param_shape_across_processes
[rank1]: [rank1]:     return dist._verify_params_across_processes(process_group, tensors, logger)
[rank1]: [rank1]: torch.distributed.DistBackendError: NCCL error in: ../torch/csrc/distributed/c10d/NCCLUtils.hpp:275, unhandled system error (run with NCCL_DEBUG=INFO for details), NCCL version 2.20.5
[rank1]: [rank1]: ncclSystemError: System call (e.g. socket, malloc) or external library call failed or device error. 
[rank1]: [rank1]: Last error:
[rank1]: [rank1]: socketStartConnect: Connect to 172.16.5.82<33513> failed : Software caused connection abort
W0401 17:27:46.576103 140548822046528 torch/distributed/elastic/multiprocessing/api.py:858] Sending process 1022770 closing signal SIGTERM
E0401 17:27:46.892293 140548822046528 torch/distributed/elastic/multiprocessing/api.py:833] failed (exitcode: 1) local_rank: 1 (pid: 1022771) of binary: /home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/bin/python3.8
Traceback (most recent call last):
  File "/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/bin/torchrun", line 8, in <module>
    sys.exit(main())
  File "/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 348, in wrapper
    return f(*args, **kwargs)
  File "/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/torch/distributed/run.py", line 901, in main
    run(args)
  File "/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/torch/distributed/run.py", line 892, in run
    elastic_launch(
  File "/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 133, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/ubuntu/tenerife/mambaforge/envs/lungAmbpy3/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 264, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
mainBMY0.py FAILED
------------------------------------------------------------
Failures:
  <NO_OTHER_FAILURES>
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2025-04-01_17:27:46
  host      : medicine-radiomics
  rank      : 1 (local_rank: 1)
  exitcode  : 1 (pid: 1022771)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
